src/electron/process_manager.ts

```diff
@@ -299,6 +301,13 @@ class ChildProcessHelper {
       }
     };
 
+    if (process.env.OUTLINE_DEBUG === 'true') {
+      this.process.stdout.on(
```

Comment:
Here is a better way to pipe the streams: https://github.com/Jigsaw-Code/outline-server/blob/48e52787a864b3d2879d3a31dd28597ba2ae0d28/src/shadowbox/server/outline_shadowsocks_server.ts#L106

Reply:
Done in 47243c809f9204c1edbd10332f40811b67a87c20